### PR TITLE
Patterns Documentation update

### PR DESCRIPTION
**Pull request by DaniMarinov**
In this update has been updated the documentation with the following patterns:
- Bar
- Data Visualizations
- Grid
- Hierarchical Grid
- Icon + Title
- Lists
- Menu
- Onboarding
- Pricing Plan
- Scheduler
- Tiles
- Tree Grid

### CODE REVIEW

**Comment by StefanIvanov** [comment]
Add "new" tags in the Table of Contents for the new and updated topics

File: en/patterns/data-visualizations.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Data Visualizations - Design System Pattern
+_description: Data Visualizations Pattern Symbol provides visualization of data or information in various layouts and representations.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Data Visualizations
+
+Use the Data Visualizations Pattern to present information and data in various chart ways, such as Area, Column,Line, Point, Spline chart.
```
Comment (StefanIvanov) [inline en/patterns/data-visualizations.md]: I'd rather say "various visual ways" or something like that, chart sounds out of place here as it is not an adjective

Comment (StefanIvanov) [reply en/patterns/data-visualizations.md]: Also mention that the pattern makes it possible to flip though different series types for example by choosing from the drop down.

File: en/patterns/icon-title.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (StefanIvanov) [inline en/patterns/icon-title.md]: I'd merge this topic into the details topic.

File: en/patterns/icon-title.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title: Icon + Title - Design System Pattern
+_description: The Icon + Title Pattern Symbols combine an Avatar with Icon or Image, Title and Body Text to display an element part of a set of components used for representation of various scenarios, such as product feature description and single step of wizard sequence.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Avatar + Badge
```
Comment (StefanIvanov) [inline en/patterns/icon-title.md]: This heading is incorrect the pattern is different

File: en/patterns/icon-title.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Use the Icon + Title Pattern to present a single element from a set of elements through avatar with featured icon or image, title and rich-text description. It can be used for various scenarios, such as product feature description and single step of wizard sequence.
+
+<img class="responsive-img" src="../images/icon_title.png" srcset="../images/icon_title@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/patterns/icon-title.md]: You had a very good point about using a set of this pattern to define the features of a product. Make the image like that.

File: en/patterns/grid.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- Grid Pattern with Summaries, Column Moving and Paging
+- Grid Pattern with Editing and Filter Row
+- Grid Pattern with Excel Style Filter, Sorting and Display Density
+- Grid Pattern with Group By, Column Resizing and Summaries
+- Grid Pattern with Pinning, Row Selection and Active Cell
+- Grid Pattern with Toolbar, Export, Row Editing and Column Hiding
+
+
+Below is illustrated the Grid Pattern with Excel Style Filter, Sorting and Display Density
+<img class="responsive-img" src="../images/grid.png" srcset="../images/grid@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/patterns/grid.md]: Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.

File: en/patterns/hierarchical-grid.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- Pinning
+- Row Selection
+- Row Editing
+- Active Cell
+- Export
+- Toolbar
+
+All Grid features are applicable to each child and parent Grid.
+
+<img class="responsive-img" src="../images/hierarchical_grid.png" srcset="../images/hierarchical_grid@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/patterns/hierarchical-grid.md]: Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.

File: en/patterns/scheduler.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+title: Scheduler - Design System Pattern
+_description: The Scheduler Pattern Symbols is a complex combination of various components to provide meaningful application scenarios.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Scheduler
+
+Use the Scheduler Symbol as presets to display Scheduler as part of project management, task management, meeting planner or e-mail application.
```
Comment (StefanIvanov) [inline en/patterns/scheduler.md]: "as a preset" sounds better and maybe we should mention that this is just an interface illustrating a scheduler without any of the scheduling functionality that is also necessary.

File: en/patterns/tiles.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+title: Tiles - Design System Pattern
+_description: The Tiles Pattern Symbols provide meaningful application scenarios for visual representation of set of products or data dashboard.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Tiles
+
+Use the Tiles Symbol as presets to display a overview of products with some of their assets, such as images and product names, as well as application features, recent items, list of items or as a rich-data dashboard.
```
Comment (StefanIvanov) [inline en/patterns/tiles.md]: "an overview"

File: en/patterns/tiles.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+<img class="responsive-img" src="../images/tiles.png" srcset="../images/tiles@2x.png 2x" />
+
+
+> [!WARNING]
+> After inserting the Tiles available as a Pattern, you should trigger `Detach from Symbol` to break it down to the Components that are used to create the layout in order to be able to generate it as Angular code. The individual Scheduler Elements, as well as the background and data binding layers must stay intact and not be detached!
```
Comment (StefanIvanov) [inline en/patterns/tiles.md]: you definitely don't mean Scheduler Elements here ;) maybe Tiles Elements

File: en/patterns/tree-grid.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+- Pinning
+- Row Selection
+- Row Editing
+- Active Cell
+- Export
+- Toolbar
+
+All Grid features are applicable to the Tree Grid Pattern.
+
+<img class="responsive-img" src="../images/tree_grid.png" srcset="../images/tree_grid@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/patterns/tree-grid.md]: Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.

**Comment by DaniMarinov** [comment]
@StefanIvanov thanks, I've updated the documentation according to your suggestions.

File: en/patterns/details.md [hunk 11]
```diff
@@ -12,17 +12,24 @@ Use the Details Pattern to display detailed information as a full-screen page us
 
 The Details Pattern comes with the styling flexibility provided by the Badges, Buttons, Tabs, and Text elements that constitute it.
 
-### Content Types
+### Details: Icon + Title
```
Comment (StefanIvanov) [inline en/patterns/details.md]: Remove the Details: from here and from the paragraph. Leave it just to say Icon + Title

File: en/patterns/grid.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
+---
+title: Grid - Design System Pattern
+_description: The Grid Pattern Symbol represents a full-blown Grid Component.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid
+
+Use the Grid Pattern symbol as an alternative starting point when designing a tabular data layout. It provides a full-blown contentwise and stylewise Grid Component that comes with all the Grid functionality, provided in the Components Library.
```
Comment (StefanIvanov) [inline en/patterns/grid.md]: Make it so "Use any of the Grid Pattern symbols as an alternative..." this implies there is more than what we show and is confirmed by the bullet list at the bottom.

File: en/patterns/hierarchical-grid.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
+---
+title: Hierarchical Grid - Design System Pattern
+_description: The Hierarchical Grid Pattern Symbol represents a parent-level Grid with a nested child-Grid inside.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Hierarchical Grid
+
+Use the Hierarchical Grid Pattern symbol as a starting point when designing a complex Hierarchical Grid with nested child Grids and some of its features. It provides a parent Grid and nested child Grid with a selected cell and visual representation of the connection between the parent and child grids. On Hierarchical Grid can be applied all the Grid functionality, provided in the Components Library.
```
Comment (StefanIvanov) [inline en/patterns/hierarchical-grid.md]: the cell is not selected but it is active because it is in focus

File: en/patterns/lists.md [hunk 14]
```diff
@@ -1,32 +1,49 @@
...
 ---
 
 ## Lists
 
-> [!Note]
-> With Indigo.Design UI Kits version 1.1.0 the patterns under Lists are no longer available due to an issue in Sketch 52. The issue is related to the many changes Sketch 52 brings along, which ultimately causes the Sketch app to crash. Once the Sketch team resolves this issue, we will push an update to the patterns library including the lists that were available before. In the mean time you can use the List Component and template and style any type of list that you would like as the Components have been fully tested and work fine.
-
-Use the Lists Pattern Symbols as presets for organized collections of data depicting common scenarios such as filterable settings, people, song playlists, searchable lists, and settings.
+Use the Lists Pattern Symbols as presets for organized collections of data in horizontal or vertical layout depicting common scenarios such as people, social media stories, multilevel list elements, settings, recently used items, products, categories of items.
```
Comment (StefanIvanov) [inline en/patterns/lists.md]: This is an old issue but can you change multilevel to multiline because we mean multiple label and lines of text :)

Comment (StefanIvanov) [reply en/patterns/lists.md]: also add the note that illustrations nested inside the Pattern Symbols cannot be code generated like you have it in the onboarding topic

File: en/patterns/form.md [hunk 15]
```diff
@@ -18,6 +18,8 @@ The Log In Forms come in a "simpler" layout with meaningful content and actions,
 
 <img class="responsive-img" src="../images/form_login-simple.png" srcset="../images/form_login-simple@2x.png 2x" />
 <img class="responsive-img" src="../images/form_login-social.png" srcset="../images/form_login-social@2x.png 2x" />
+<img class="responsive-img" src="../images/form_illustrated_forgot_password.png" srcset="../images/form_illustrated_forgot_password@2x.png 2x" />
+<img class="responsive-img" src="../images/form_illustrated_account_type.png" srcset="../images/form_illustrated_account_type@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/patterns/form.md]: Add the note that illustrations nested inside the Pattern Symbols cannot be code generated like you have it in the onboarding topic

**Pull request merged**